cluster/cluster_test.go

```diff
@@ -135,3 +139,22 @@ func TestUnsupportedVersion(t *testing.T) {
 	err = json.Unmarshal([]byte(`{"cluster_definition":{"version":"invalid"}}`), &lock)
 	require.ErrorContains(t, err, "unsupported definition version")
 }
+
+// TestExamples tests whether charon is backwards compatible with all examples.
+func TestExamples(t *testing.T) {
+	lockFiles, err := filepath.Glob("examples/*lock*")
```

Comment:
added `/examples` folder which contains real world examples that we need to support